shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -970,6 +985,25 @@ public boolean shouldAttachEngineToActivity() {
     return true;
   }
 
+  /**
+   * Whether to handle the deeplinking from the {@code Intent} automatically if the {@code
+   * getInitialRoute} returns null.
+   *
+   * <p>The default implementation looks for the value of the key `flutter_handle_deeplinking` in
```

Comment:
Be explicit about the metadata tag. Just saying key value in an AndroidManifest will be very hard to decipher.